apps/asap-server/test/integration/research-outputs.integration-test.ts

```diff
@@ -51,9 +51,13 @@ describe('Research Outputs', () => {
     try {
       await createResearchOutput(researchOutput);
     } catch (e) {
-      expect(e.name).toBe('HTTPError');
-      expect(e.output.statusCode).toBe(400);
-      expect(e.data).toMatch('doi.iv: Must follow the pattern');
+      const parsedErrorData = JSON.parse(e.data);
```

Comment:
whats going on here if i may ask? why the change?

Reply:
the testing data format changed slightly, have updated test accordingly

Reply:
i was going to suggest the jest expect...toThrow syntax, same as above, but I am not 100% sure you are able to test all these different error properties nicely with it. Since, i would give it a go, even though we had try/catch in this test previously